### ISSUE TITLE
Build failure on crates + master, compiler need mismatch

### ISSUE THREAD

**Issue by amyipdev** [immunant/c2rust#332]
Context: https://github.com/gimli-rs/object/issues/285

Object's compiler bug only occurs on the nightly version used to compile c2rust. C2Rust of course can't compile on stable due to an issue with the AST builder:

```rs
error[E0463]: can't find crate for `rustc`
 --> /home/amy/.cargo/registry/src/github.com-1ecc6299db9ec823/c2rust-ast-builder-0.15.0/src/lib.rs:2:1
  |
2 | extern crate rustc;
  | ^^^^^^^^^^^^^^^^^^^ can't find crate

error: aborting due to previous error

For more information about this error, try `rustc --explain E0463`.
error: could not compile `c2rust-ast-builder`
```

**Comment by thedataking** [comment]
Please have a look at https://github.com/immunant/c2rust/issues/323 for a possible way to compile object on the nightly version required by C2Rust.

**Comment by milahu** [comment]
with `rust-nightly-2019-12-05` and `cargo build --locked` i still get

```
   Compiling c2rust-ast-builder v0.15.0 (/build/source/c2rust-ast-builder)
error[E0463]: can't find crate for `rustc`
```

this is on `nixos linux` so this error probably appears only in clean environments

fixed in #354

**Comment by rinon** [comment]
Fixed in the v0.16.0 release
